src/felles-komponenter/skjema/skjema.less

```diff
@@ -0,0 +1,50 @@
+@coreModulePath: '../../../node_modules/';
```

Comment:
Hentet inn igjen skjema-styling. Ser at jeg tok ut litt for mye sist. Dette er styling for visning av feiloppsummering.